date_module.js

```diff
@@ -0,0 +1,88 @@
+//setdate
+// set a variable for the date using built in js date function
+// setDate is a method that allows the value of the date variable to be changed by user input
+// user enters a new date which is saved to variable
+
+var myObj = ( function iife() {
+  var myDate = new Date();
+
+  function setDate() {
+    var userInputDate;
+    if (typeof userInputDate !== 'undefined') {
+      myDate = userInputDate;
+    }
+  }
+
+  function getDate( format ) {
+  	myDate.getDate({ format: "milliseconds" });
+/*
+  	  return myDate.getMilliseconds();
+    myDate.getDate({ format: "formatted" });
+
+      return myDate.getDayName + myDate.getMonthName = myDate.getFullYear;
+    myDate.getDate();
+  */
+  }
+
+  function getDayName() {
+    var weekday = new Array(7);
+    weekday[0]=  'Sunday';
+    weekday[1] = 'Monday';
+    weekday[2] = 'Tuesday';
+    weekday[3] = 'Wednesday';
+    weekday[4] = 'Thursday';
+    weekday[5] = 'Friday';
+    weekday[6] = 'Saturday';
+
+    return weekday[myDate.getDay()];
+  }
+
+  function getMonthName() {
+    var month = new Array();
```

Comment:
Same deal here as with the weekday array - use an array literal rather than the array constructor.